prep-html-skeleton/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+</html>
```

Comment:
The closing tag for the `html` element should be the last thing in your HTML document. This element is being closed too soon.